recipes/physo/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "physo" %}
+{% set version = "1.1.9" %}
+{% set python_min = "3.9" %}
```

Comment:
```suggestion
```

Just remove this unless at some time in the future, the minimum supported python version is greater than:
https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/conda_build_config.yaml#L828-L831